picus/algorithms/dpvl.rkt

```diff
@@ -421,6 +413,11 @@
   ; computing rcdmap need no ab0 lemma from optimization phase 0
   (set! :sdmcnsts (:normalize-r1cs (:expand-r1cs :cnsts)))
 
+  ; generate linear-clauses requires no optimization to exclude ror and rand
```

Comment:
The comment here doesn't make sense. What is the point of it?

Reply:
Well, this comment was moved from somewhere else:

https://github.com/Veridise/Picus/pull/18/files#diff-5686555b4402abd7c51e61ada773d8a9d2f6cf5b8516a6325a65fd6e4e4cd812L196

I think it's meant to point out why we are using `:sdmcnsts` as opposed to optimized constraints.

Reply:
Oh i see so it doesn't work when the constraints are compiled via -ON

Reply:
This is not about `-ON` compilation. It's about our internal passes that simplify R1CS. See https://github.com/Veridise/Picus/tree/main/picus/optimizers